### PR TITLE
chore: adding initial repo files

### PR DESCRIPTION
**Pull request by kupermind**
- Initial files.

### CODE REVIEW

**Review by 77ph — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
-# autonolas-staking-programmes
-Autonolas Staking Programmes
+# Autonolas Staking Programmes
+
+## Introduction
+This repository contains the Autonolas Staking Programmes set of contracts. It is built on top of a core service staking
+functionality of [autonolas-staking-programmes](https://github.com/valory-xyz/autonolas-staking-programmes).
```
Comment (mariapiamo) [inline README.md:5]: do you mean registries here?

**Review by mariapiamo — COMMENTED** [review]
I would suggest to rename contracts subfolder "p1" with "Staking contracts checking Mech Usage" or similar

File: README.md (hunk 1 above)
Comment (kupermind) [reply README.md:5]: Yes, good catch, it was registries before, but the renaming changed it :)

File: abis/0.8.21/ServiceStakingMechUsage.json [hunk 2]
```diff
@@ -0,0 +1,1054 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "ServiceStakingMechUsage",
+  "sourceName": "contracts/p1/ServiceStakingMechUsage.sol",
```
Comment (mariapiamo) [inline abis/0.8.21/ServiceStakingMechUsage.json:4]:
```suggestion
  "sourceName": "contracts/mech_usage/ServiceStakingMechUsage.sol",
```

File: abis/0.8.21/ServiceStakingTokenMechUsage.json [hunk 3]
```diff
@@ -0,0 +1,1157 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "ServiceStakingTokenMechUsage",
+  "sourceName": "contracts/p1/ServiceStakingTokenMechUsage.sol",
```
Comment (mariapiamo) [inline abis/0.8.21/ServiceStakingTokenMechUsage.json:4]:
```suggestion
  "sourceName": "contracts/mech_usage/ServiceStakingTokenMechUsage.sol",
```

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**